dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityQueryParams.java

```diff
@@ -186,19 +186,19 @@ public TrackedEntityQueryParams() {}
    */
   public void handleOrganisationUnits() {
```

Comment:
Maybe not part of this PR but this method could be moved to `TrackedEntityOperationParamsMapper` so we don't need to change orgUnits in `TrackedEntityQueryParams`?

Reply:
I agree. It should be the same with these three [validation methods](https://github.com/dhis2/dhis2-core/pull/15356/files#diff-069b27403f4f267ecb5e14378a2e9c32d339e92acb494e688b821df93de33acbR343), they should probably be moved to the mapper as well.

In this PR I'm rolling back some of the changes I did, and since this is already hard enough, I didn't want to add more complexity to it by refactoring the operations layer. I'll create a new ticket to deal with these validations.